includes/cli/anonymize.php

```diff
@@ -19,10 +21,27 @@ class SafetyNet_CLI extends WP_CLI_Command {
 	*
 	*/
 	public function anonymize( $args ) {
+
+		$info = WP_CLI::colorize( '%pThis process will anonymize your current users, orders and customers with dummy data.%n ' );
+		WP_CLI::log( $info );
+
+		WP_CLI::warning( 'Please proceed with caution if you have a site with a large number of users/orders/customers' );
+
+		WP_CLI::confirm( 'Are you sure you want to do this?' );
+
+		WP_CLI::log( '- Copying users to temporary tables ...' );
+		copy_and_clear_user_tables();
+
+		WP_CLI::log( '- Anonymizing users ... ' );
 		anonymize_users();
 
+		WP_CLI::log( '- Storing the anonymized users to the default tables ... ' );
+		store_anonymized_user_data();
```

Comment:
Each of these steps definitely needs some more verbosity. A nice success message if everything goes to plan, and a detailed error output if not ideally.